README.md

```diff
@@ -386,6 +386,34 @@ $ helm install my-release \
     neotys/nlweb
 ```
 
+#### Side-car containers (experimental)
+
+> [!NOTE]
+> **This feature is experimental.**
+> It allows to deploy side-car containers on the frontend and backend Deployments.
+
+> [!CAUTION]
+> When using `services.<name>.*` values the `<name>` must map the name of a port declared in containers under `extra.containers.backend` or `extra.containers.frontend`.
+
+For a detailed example on how to use this feature take a look at [this example](./doc/sidecar-example/README.md).
+
+
+Parameter | Description | Default
+----- | ----------- | -------
+`hostOverrides.webapp` | Overrides the app configuration if the hostname used to access NeoLoad Web Frontend is different than the value of `services.webapp.host`|
+`hostOverrides.api` | Overrides the app configuration if the hostname used to access NeoLoad Web API is different than the value of `services.api.host`|
+`hostOverrides.files` | Overrides the app configuration if the hostname used to access NeoLoad Web Files API is different than the value of `services.files.host`|
+`extra.containers.backend` | Allows to specify a list of valid [Containers](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.26/#container-v1-core). They will be appended to the list of Containers of the backend Deployment. |
+`extra.containers.frontend` | Allows to specify a list of valid [Containers](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.26/#container-v1-core). They will be appended to the list of Containers of the frontend Deployment. |
+`extra.volumes.frontend` | Allows to specify a list of valid [Volumes](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.26/#volume-v1-core). They will be to the PodSpec of the frontend Deployment. |
+`extra.volumes.backend` | Allows to specify a list of valid [Volumes](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.26/#volume-v1-core). They will be to the PodSpec of the backend Deployment. |
+`services.<name>.component` | The Deployment to which this service attach it can be `frontend` or `backend` | 
```

Comment:
[GPT] The Deployment to which this service is attached can be either frontend or backend.